### DISCUSSION TITLE
Alexa (Haaska) funzionamento a singhiozzo

### DISCUSSION THREAD

**Discussion by michele-diurno** [indomus/forum#3183] [HUB: Home Assistant]
Ciao a tutti,
volevo chiedervi se capita anche a voi che i comandi impartiti ad Alexa a volte vengono recepiti e a volte no, delle volte devo ripeterli una seconda volta per attivarli.
Perchè funziona a singhiozzo? La rete a me sembra stabile, ma sembra che va in standby qualcosa e poi al risveglio comincia a funzionare.
Non so se mi sono spoiegato bene, ma i comandi vengono recepiti in modo discontinuo. Cosa posso controllare? Questo succede su tutti i dispositivi in casa.
Grazie a tutti.

**Comment by KC-inDomus** [comment]
sei su duckdns, con home assistant?

**Comment by michele-diurno** [comment]
Si

**Reply by KC-inDomus** [reply]
9 su 10 è per quello. migra a clouduflare

**Comment by michele-diurno** [comment]
Ok, grazie per il suggerimento. Ho provato nel frattempo ad impostare ip statico su tutti gli shelly, sembra essere migliorata la situazione nel senso che i comandi funzionano 9 volte su 10, ma il tempo di risposta dal comando vocale ad Alexa alla risposta dello Shelly (es. tapparella) è di 2/3 secondi. 
Insomma ancora non ci siamo.
Per cloudfare dovrò riprendere la guida di indomus? Lo chiedo perchè è stato un mezzo incubo riuscire a completare la guida, sbagliavo sempre qualcosa ...

**Reply by KC-inDomus** [reply]
fissare gli IP non serve a nulla. per migrare usa [questa](https://indomus.it/guide/come-migrare-home-assistant-da-duckdns-a-cloudflare/), dovresti riuscire

**Comment by michele-diurno** [comment]
Si ti confermo che gli ip statici non hanno risolto il problema, questa mattina ho chiesto ad Alexa di alzare la tapparella, lei ha confermato ma la tapparella è rimasta ferma. Spero che con Cloudfare migliori la situazione. Nel frattempo sempre grazie per l'aiuto. Per fortuna esistono questi forum.

**Reply by KC-inDomus** [reply]
u welcome. vedrai che la stabilità offerta dal tunnelling non ti darà più problemi di raggiungibilità.

**Comment by michele-diurno** [comment]
Utilizzo Ngnix anzichè Caddy potrebbe essere questo il problema del funzionamento a singhiozzo o è sempre rinconducibile a Duckdns? Scusate la mia ignoranza ma questa materia è ostica in quanto solo nei ritagli di tempo ed in modo discontinuo riesco a cimentarmi. 
Sempre grazie.

**Reply by KC-inDomus** [reply]
no, il problema è duckdns.

**Comment by michele-diurno** [comment]
ok grazie mille.
